admin/class-yoast-form.php

```diff
@@ -616,14 +635,14 @@ public function radio( $var, $values, $legend = '', $legend_attr = [] ) {
 	 *
 	 * @since 3.1
 	 *
-	 * @param string $var    The variable within the option to create the radio buttons for.
-	 * @param array  $values Associative array of on/off keys and their values to be used as
-	 *                       the label elements text for the radio buttons. Optionally, each
-	 *                       value can be an array of visible label text and screen reader text.
-	 * @param string $label  The visual label for the radio buttons group, used as the fieldset legend.
-	 * @param string $help   Inline Help that will be printed out before the visible toggles text.
+	 * @param string                 $var    The variable within the option to create the radio buttons for.
+	 * @param array                  $values Associative array of on/off keys and their values to be used as
+	 *                                       the label elements text for the radio buttons. Optionally, each
+	 *                                       value can be an array of visible label text and screen reader text.
+	 * @param string                 $label  The visual label for the radio buttons group, used as the fieldset legend.
+	 * @param WPSEO_Admin_Help_Panel $help   Inline Help that will be printed out before the visible toggles text.
 	 */
-	public function toggle_switch( $var, $values, $label, $help = '' ) {
+	public function toggle_switch( $var, $values, $label, $help = null ) {
```

Comment:
You are changing public interface here. For BC reasons I would love to support the help as string as well.